test/components/form-attachment/list.spec.js

```diff
@@ -1037,15 +1037,31 @@ describe('FormAttachmentList', () => {
   });
 
   describe('dataset linking', () => {
-    beforeEach(() => {
+    it('sends a request for datasets if the project has one', () => {
+      testData.extendedProjects.createPast(1, { forms: 2, datasets: 1 });
+      testData.extendedDatasets.createPast(1);
       testData.extendedForms.createPast(1, { draft: true });
```

Comment:
There used to be a `beforeEach()` hook that created a form draft. However, I think it's better to wait to create the form draft until the project is created. Otherwise, creating the form draft will create a separate project.